src/components/Skeleton/AlertProvider.jsx

```diff
@@ -8,16 +8,38 @@ import DottedLine from "components/common/DottedLine";
 
 import WbIncandescentRoundedIcon from "@material-ui/icons/WbIncandescentRounded";
 
+const useDelay = (isOpen, delayTime, type) => {
+  const [value, setValue] = useState(false);
+
+  useEffect(() => {
+    let timeoutId;
+    if (isOpen && !value) {
+      if (type === "mount") setValue(true);
+      else timeoutId = setTimeout(() => setValue(true), delayTime);
+    } else if (!isOpen && value) {
+      if (type === "mount")
+        timeoutId = setTimeout(() => setValue(false), delayTime);
+      else setValue(false);
+    }
+    return () => clearTimeout(timeoutId);
+  }, [isOpen]);
+  return value;
+};
+
 const AlertProvider = () => {
   const messageCache = useRef("");
   const [message, setMessage] = useRecoilState(alertAtom);
 
+  const shouldMount = useDelay(!!message, 150, "mount");
+  const shouldDisplay = useDelay(!!message, 0, "display");
+
   const onClickClose = () => setMessage(null);
   if (message) messageCache.current = message;
 
+  if (!shouldMount) return null;
   return (
     <Modal
-      display={message ? true : false}
+      display={shouldDisplay}
```

Comment:
```suggestion
      display={!!message}
```

`shouldDisplay` Hook은 delay 파라미터가 0이니 없애고,
위 처럼 사용해도 되는 거 아닌가요??

Reply:
그리고 `useDelay` Hook에서 `type` 파라미터 지우고,
`delayTime` 파라미터는 default값 `theme.duration`으로 사용하는 것 어떠신가요?

Reply:
> `shouldDisplay` Hook은 delay 파라미터가 0이니 없애고, 위 처럼 사용해도 되는 거 아닌가요??

`shouldMount`가 `true`가 되기 전에 display가 되어 버려서 저렇게 shouldDisplay도 콜스택에 넣어야 창이 뜰때도 애니메이션이 되는것 같습니다

Reply:
@imYourChoi 제가 작업한 후 이 브랜치로 PR 날려서 제안드려도 될까요?

Reply:
좋습니다!